src/main/java/Array2.java

```diff
@@ -0,0 +1,18 @@
+public class Array2 {
+
+    public int sumOfThreeNumberMax(int[] arr, int n) {
+
+        int max = arr[0] + arr[1] + arr[2];
+        for (int i = 0; i < n - 2; ++i) {
+            int sumTemp = arr[i];
```

Comment:
Em sắp xếp trước, sau đó gắp ra 3 số bự nhất ở đầu hoặc cuối dãy (tùy order sắp xếp)  rồi cộng lại là xong

Reply:
em cũng nghĩ đến cách đó nhưng sẽ tốn thêm công sắp xếp, nếu là sắp xếp nổi bọt thường thì cũng tốn 2 vòng for như cách em đang làm.